spark/src/test/scala/org/apache/comet/CometArrayExpressionSuite.scala

```diff
@@ -425,7 +425,6 @@ class CometArrayExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelp
           checkSparkAnswerAndOperator(
             sql("SELECT array_repeat(_2, 5) from t1 where _2 is not null"))
           checkSparkAnswerAndOperator(sql("SELECT array_repeat(_2, 5) from t1 where _2 is null"))
-          checkSparkAnswerAndOperator(sql("SELECT array_repeat(_3, _4) from t1 where _3 is null"))
```

Comment:
the average value of `_4` is 5,000 so for 10k rows we are repeating values a total of 50m times